example/TS/src/DistributedHelloWorld/HelloWorld.lf

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
I think if this example was turned into a test, it might be less of a headache to keep whatever is already working intact as we make changes to federated execution.

Reply:
Thank you for the great suggestion. I tried to turn this into a test, but realized that we need to make significant changes to the TypeScript code generation and the testing process, to support the federated test for the TypeScript target. I planned out how to enable federated tests in TypeScript in my comment [here](https://github.com/lf-lang/lingua-franca/issues/451#issuecomment-928521464).

Do you think you could approve this pull request for now? I'm afraid that if the pull request is left for too long, it might be harder to merge later.